### PR TITLE
Update document type codelist

### LINKED ISSUE #261: documentType codelist: Add 'projectFeasibilityStudy' code
The reactive disclosure section of the CoST IDS includes the following element:

> Project brief or Feasibility study

## Proposal

Add a 'projectFeasibilityStudy' code to the `documentType` codelist with the following definition:

> Documentation of feasibility studies carried out for this project, providing information on net benefits or costs of the project.

Alternatively, we could add "or project" to the description of the existing (contracting process) 'feasibilityStudy' code from OCDS:

> Documentation of feasibility studies carried out for this contracting process **or project**, providing information on net benefits or costs of the proposed goods, works or services.

@jpmckinney which do you think is the best option?

### LINKED ISSUE #263: documentType codelist: Add 'resettlementPlan' code
The reactive disclosure section of the CoST IDS includes the following element:

> Resettlement and compensation plan

The IFC [defines](https://www.ifc.org/wps/wcm/connect/ee19f150-f505-41db-891f-6ef5557195b6/ResettlementHandbook.PDF?MOD=AJPERES&CACHEID=ROOTWORKSPACE-ee19f150-f505-41db-891f-6ef5557195b6-jkD0CRL) (page 12) a resettlement action plan as:

> The document in which a  project sponsor or other responsible entity specifies the procedures that it will follow and the actions that it will take to mitigate adverse effects, compensate losses, and provide development benefits to persons and communities affected by an investment project.

## Proposal

Add a 'resettlementPlan' code to the `documentType` codelist with the following title and description:

> **Resettlement and compensation plan**
> Documentation of the procedures and actions to mitigate adverse effects, compensate losses, and provide development benefits to persons and communities affected by the contracting process or project.

### LINKED ISSUE #265: documentType codelist: add 'projectFinancialAgreement' code
The reactive disclosure section of the CoST IDS includes the following element:

> **Financial agreement**
> Loan or donation agreement with financing conditions

## Proposal

Add a 'projectFinancialAgreement' code to the `documentType` codelist with the following title and description:

> **Project financial agreement**
> Loan or donation agreement with financing conditions

Alternatively, we could omit "project" but we might then find there are other types of financial agreement that do not fit the description.

### LINKED ISSUE #266: documentType codelist: Add 'budgetAmendmentDecision' code
The reactive disclosure section of the CoST IDS includes the following element:

> **Budget amendment decision**
> Document accrediting the approval of budget expansion or extension for this project

## Proposal

Add a 'budgetAmendmentDecision' code to the `documentType` codelist with the following title and description:

> **Budget amendment decision**
> Document accrediting the approval of budget expansion or extension for this project

### LINKED ISSUE #267: documentType codelist: Add 'projectCompletionReport' code
The reactive disclosure section of the CoST IDS includes the following element:

> **Project completion report**
> A certificate of completion issued by the relevant authority demonstrating that the works are completed with a certain level of quality (Act of reception of works).

## Proposal

Add a 'projectCompletionReport' code to the `documentType` codelist with the following title and description:

> **Project completion report**
> A certificate of completion issued by the relevant authority demonstrating that the works are completed with a certain level of quality, also known as the act of reception of works.

### LINKED ISSUE #268: documentType codelist: add 'technicalAuditReport' code
The reactive disclosure section of the CoST IDS includes the following element:

> **Technical audit report**
> Documentation of checks that construction work has been undertaken as specified

## Proposal

Add a 'technicalAuditReport' code to the `documentType` codelist with the following title and description:

> **Technical audit report**
> Documentation of checks that construction work has been undertaken as specified

### LINKED ISSUE #269: documentType codelist: Add 'financialAuditReport' code
The reactive disclosure section of the CoST IDS includes the following element:

> **Financial audit report**
> Documentation of checks that project financial statements are correct and complete

## Proposal

Add a 'financialAuditReport' code to the `documentType` codelist with the following title and description:

> **Financial audit report**
> Documentation of checks that project financial statements are correct and complete

### LINKED ISSUE #272: documentType codelist: add 'qualityAssuranceReport' code
The reactive disclosure section of the CoST IDS includes the following element:

> **Quality assurance reports**
> Documentation identifying deficiencies in designs, specifications, standards, methods, procedures and practices, and identifying best practices.

## Proposal

Add a 'qualityAssuranceReport' code to the `documentType` codelist with the following title and description:

> **Quality assurance report**
> Documentation identifying deficiencies in designs, specifications, standards, methods, procedures and practices, and identifying best practices.

### LINKED ISSUE #274: documentType codelist: Add 'incorporationCertificate' code
The reactive disclosure section of the CoST IDS has the following element:

> Registration and ownership of firms

CoST clarified this requirement covers both data and documents. The requirement can be met by providing an organization identifier (data) and a copy of the certificate of incorporation (document).

## Proposal

Add a 'incorporationCertificate' code to the `documentType` codelist, with the following title and description:

> **Certificate of incorporation**
> An official document that proves that a company has been legally created and officially exists.

### LINKED ISSUE #275: documentType codelist: Add 'contractAmendment' code
The reactive disclosure section of the CoST IDS has the following element:

> **Contract amendments**
> Documentation of the amended contract terms.

## Proposal

Add a 'contractAmendment' code to the `documentType` codelist, with the following title and description:

> **Contract amendment**
> Documentation of the amended contract terms.

### PR DESCRIPTION
**Pull request by pindec**
fixes #261 , fixes #263 , fixes #265 , fixes #266 , fixes #267 , fixes #268 , fixes #269 , fixes #272 , fixes #274 , fixes #275 

One small question - should we amend the source of 'feasibilityStudy' and 'completionCertificate' since we have amended the text of the OCDS code descriptions?

### CODE REVIEW

**Comment by duncandewhurst** [comment]
> One small question - should we amend the source of 'feasibilityStudy' and 'completionCertificate' since we have amended the text of the OCDS code descriptions?

Yes, let's do that, in keeping with the modified 'needsAssessment' code from OCDS.

**Review by duncandewhurst — APPROVED** [review]
Thanks @pindec, looks good to me. Please request a review from @jpmckinney once you've updated the source for the 'feasibilityStudy' and 'completionCertificate' codes.

File: docs/reference/changelog.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+* [#263](https://github.com/open-contracting/infrastructure/issues/263) add 'resettlementPlan' code to documentType codelist
+* [#265](https://github.com/open-contracting/infrastructure/issues/265) add 'financialAgreement' code to documentType codelist
+* [#266](https://github.com/open-contracting/infrastructure/issues/266) add 'budgetAmendmentApproval' code to documentType codelist
+* [#267](https://github.com/open-contracting/infrastructure/issues/267) amend 'completionCertificate' code description to include project
+* [#268](https://github.com/open-contracting/infrastructure/issues/268) add 'technicalAuditReport' code to documentType codelist, remove 'finalAudit'
+* [#269](https://github.com/open-contracting/infrastructure/issues/269) add 'financialAuditReport' code to documentType codelist
+* [#272](https://github.com/open-contracting/infrastructure/issues/272) add 'qualityAssuranceReport' code to documentType codelist
+* [#274](https://github.com/open-contracting/infrastructure/issues/274) add 'incorporationCertificate' code to documentType codelist
+* [#275](https://github.com/open-contracting/infrastructure/issues/275) add 'contractAmendment' code to documentType codelist
```
Comment (jpmckinney) [inline docs/reference/changelog.md]: Make the changelog easier to scan:

```suggestion

#### documentType codelist

Changed:

* [#261](https://github.com/open-contracting/infrastructure/issues/261) Update description of 'feasibilityStudy' code to include "project"
* [#267](https://github.com/open-contracting/infrastructure/issues/267) Update description of 'completionCertificate' code to include "project"

Added:

* [#263](https://github.com/open-contracting/infrastructure/issues/263) 'resettlementPlan'
* [#265](https://github.com/open-contracting/infrastructure/issues/265) 'financialAgreement'
* [#266](https://github.com/open-contracting/infrastructure/issues/266) 'budgetAmendmentApproval'
* [#268](https://github.com/open-contracting/infrastructure/issues/268) 'technicalAuditReport'
* [#269](https://github.com/open-contracting/infrastructure/issues/269) 'financialAuditReport'
* [#272](https://github.com/open-contracting/infrastructure/issues/272) 'qualityAssuranceReport'
* [#274](https://github.com/open-contracting/infrastructure/issues/274) 'incorporationCertificate'
* [#275](https://github.com/open-contracting/infrastructure/issues/275) 'contractAmendment'

Removed:

* [#268](https://github.com/open-contracting/infrastructure/issues/268) 'finalAudit'
```

File: docs/reference/changelog.md [hunk 2]
```diff
@@ -1,23 +1,45 @@
...
 
+* [#217](https://github.com/open-contracting/infrastructure/issues/217) - remove repeated 'OCDS:' in mapping documentation
 * [#268](https://github.com/open-contracting/infrastructure/issues/268), [#269](https://github.com/open-contracting/infrastructure/issues/269) replace 'finalAudit' with 'technicalAuditReport' and 'financialAuditReport' in mapping
 
+### Schema
+
+* [#277](https://github.com/open-contracting/infrastructure/issues/277) - add `forecasts` and `metrics` which can be used to publish implementation progress reports
+* [#282](https://github.com/open-contracting/infrastructure/pull/282) - update fields shared with OCDS for PPPs 1.0.0-beta2 and OCDS 1.1.4.
+
+### Codelists
```
Comment (jpmckinney) [inline docs/reference/changelog.md]: There is already a codelists heading below.

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**